addons/info.go

```diff
@@ -7,31 +7,61 @@ import (
 	"context"
 	"fmt"
 	"os"
+	"slices"
 	"strings"
 	"time"
 
 	"github.com/olekukonko/tablewriter"
-	"gopkg.in/errgo.v1"
+
+	"github.com/Scalingo/go-utils/errors/v2"
 
 	"github.com/Scalingo/cli/config"
+	"github.com/Scalingo/cli/db"
 	"github.com/Scalingo/cli/utils"
+	"github.com/Scalingo/go-scalingo/v6"
 )
 
 // Info is the command handler displaying static information about one given addon
 func Info(ctx context.Context, app, addon string) error {
 	c, err := config.ScalingoClient(ctx)
 	if err != nil {
-		return errgo.Notef(err, "fail to get Scalingo client")
+		return errors.Wrap(ctx, err, "get Scalingo client")
 	}
 
 	addonInfo, err := c.AddonShow(ctx, app, addon)
 	if err != nil {
-		return errgo.Notef(err, "fail to get addon information")
+		return errors.Wrap(ctx, err, "get addon information")
+	}
+
+	isDatabase := slices.ContainsFunc(db.SupportedDatabases, func(s string) bool {
+		return strings.EqualFold(s, addonInfo.AddonProvider.ID)
+	})
+
+	dbInfo := [][]string{}
+	if isDatabase {
+		dbInfo, err = getDatabaseInfo(ctx, c, app, addon)
+		if err != nil {
+			return errors.Wrap(ctx, err, "get database information")
+		}
 	}
 
+	t := tablewriter.NewWriter(os.Stdout)
+	t.Append([]string{"Addon Provider", addonInfo.AddonProvider.Name})
```

Comment:
question: why not filled the `dbInfo` struct like the one in the `getDatabaseInfo` function?
Example:
```go
    lines := [][]string{
        {"Database Type", dbInfo.TypeName},
	...
    }